packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -51,6 +56,34 @@ export async function activate(context: ExtensionContext): Promise<void> {
     clientOptions
   );
 
+  const jsonSchemaConfig:
+    | WorkspaceConfiguration
+    | undefined = workspace.getConfiguration().get("json.schemas");
+  if (
+    process.env["THEIA_PARENT_PID"] !== undefined &&
+    isArray(jsonSchemaConfig)
+  ) {
+    const providedManifestSchemaConfigs = find(jsonSchemaConfig, (_) => {
+      return some(
```

Comment:
`some` works on an array, however there may only be a **single** fileMatch property (no array).
- https://github.com/microsoft/vscode/blob/0c25f5a737a0c4b0945783588c13cfee11d0ecb1/src/vs/workbench/api/common/jsonValidationExtensionPoint.ts#L12-L15